docs/core_docs/docs/integrations/chat/azure_ml.mdx

```diff
@@ -0,0 +1,22 @@
+# Azure Machine Learning Chat
+
+You can deploy models on Azure with the endpointUrl, apiKey, and deploymentName
+when creating the AzureMLChatParams to call upon later. You must import a ContentFormatter
+or create your own using the ChatContentFormatter interface.
+
+```typescript
```

Comment:
Prefer putting this in a `.ts` file inside `./examples/src/models/chat`. This way the example code can be built & typechecked.

Then, import it using a `CodeBlock`